src/scalarstats.jl

```diff
@@ -47,10 +47,11 @@ end
 # compute mode, given the range of integer values
 """
     mode(a, [r])
+    mode(a::AbstractArray, wv::AbstractArray{<: Real})
```

Comment:
Sorry I got this wrong! :-D
```suggestion
    mode(a::AbstractArray, wv::AbstractWeights)
```